deepcell/model_zoo/featurenet.py

```diff
@@ -446,7 +450,39 @@ def bn_feature_net_3D(receptive_field=61,
                     kernel_initializer=init, padding='valid',
                     kernel_regularizer=l2(reg))(x[-1]))
     x.append(BatchNormalization(axis=channel_axis)(x[-1]))
-    x.append(Activation('relu')(x[-1]))
+    feature = Activation('relu')(x[-1])
+
+    def __merge_temporal_features(feature, mode='conv', residual=False, n_filters=256,
+                                  n_frames=3, padding=True, temporal_kernel_size=3):
+        if mode is None:
+            return feature
+        if mode is 'conv':
```

Comment:
For string comparison, you should keep using `==`, i.e. `if mode == "conv"`.  `is` should be used for object identity. [Here is a stackoverflow post about it](https://stackoverflow.com/questions/6570371/when-to-use-and-when-to-use-is).

I also think it would be a bit cleaner to just `str.lower()` the input, so you do not have to worry about any case mismatches.  With what we have now, `LSTm` would throw an error (in some cases that could be desirable I suppose).

Either way, "conv" mode is still case sensitive.